gcp/modules/couchdb/main.tf

```diff
@@ -2,10 +2,64 @@ terraform {
   backend "gcs" {}
 }
 
+variable "couchdb_prd_replicas" {
+  default = 3
+}
+
+variable "couchdb_dev_replicas" {
+  default = 2
+}
+
+variable "release_namespace" {
+  default = "gpii"
+}
+
+variable "env" {}
 variable "secrets_dir" {}
 variable "values_dir" {}
-variable "release_namespace" {
-	default = "gpii"
+variable "project_id" {}
+variable "serviceaccount_key" {}
+
+provider "google" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
+}
+
+resource "google_compute_disk" "couchdb" {
+  count = "${var.env == "dev" ? var.couchdb_dev_replicas : var.couchdb_prd_replicas}"
+  name  = "couchdb-${count.index}"
```

Comment:
Is the PV `couchdb-0` guaranteed to be associated with the couchdb Pod `couchdb-0`? If not, I fear this is going to be pretty confusing.

The AWS side names PVs based on their AZ, which I think is helpful. Not sure if it's possible here.

Reply:
I agree this can be confusing, let me investigate possible alternatives.